keyboards/xelus/wombat/readme.md

```diff
@@ -0,0 +1,13 @@
+# Wombat
```

Comment:
Does not follow [template](https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md)